experimental/sdg-tracker/src/components/countries/CountriesContent.tsx

```diff
@@ -31,6 +32,11 @@ import {
   FulfillResponse,
 } from "../../utils/types";
 import { SearchBar } from "../layout/components";
+
+// Approximate chart heights for lazy-loading
+const CHART_HEIGHT = 389;
```

Comment:
Something to think about for later: If we ever add horizontal bar charts, that chart height is dynamic based on number of vars/places to plot and if there's a set bar height. We should remember to revisit this if horizontal bar charts get added to the UN pages.

Reply:
That sounds good- we can come up with a more dynamic chart height estimate depending on the parameters